be/src/storage/rowset/rowset_writer.h

```diff
@@ -124,6 +124,7 @@ class RowsetWriter {
 
 protected:
     vectorized::DictColumnsValidMap _global_dict_columns_valid_info;
+    const uint64_t MAX_SEGMENT_SIZE = config::max_segment_file_size;
```

Comment:
```suggestion
    const uint64_t _max_segment_size = config::max_segment_file_size;
```
This is a normal class member variable.

Reply:
BTW, why not just using `config::max_segment_file_size` directly?

Reply:
I use config::max_segment_file_size to replace MAX_SEGMENT_SIZE directly